client/components/editor/teaching-elements/Assessment/TextSelection/Range.js

```diff
@@ -0,0 +1,52 @@
+import { extractIndex } from './helpers';
+import filter from 'lodash/filter';
+import first from 'lodash/first';
+import inRange from 'lodash/inRange';
+import last from 'lodash/last';
+import sortBy from 'lodash/sortBy';
+
+export default class Range {
+  constructor(start, end) {
+    [start, end] = sortBy([start, end]);
```

Comment:
How is this different than:
```js
[start, end] = [start, end].sort();
```

Reply:
Quote from MDN for Array.sort: `The default sort order is according to string Unicode code points.` I need sort order according to values. e.g. `[19,9].sort() // => [19,9]`